nbconvert/filters/strings.py

```diff
@@ -188,14 +188,59 @@ def get_lines(text, start=None,end=None):
     # Return the right lines.
     return "\n".join(lines[start:end]) #re-join
 
-def ipython2python(code):
+def _mpl_magic_regex_generator(forbidden_backends=None):
+    """Create regex for detecting incompatible backends in mpl magics.
+    
+    Parameters
+    ----------
+
+    forbidden_backens: List
```

Comment:
Typo: "backens" -> "backends". Also, is there a reason the list of backends is called `forbidden_backends` here and `bad_backends` elsewhere?